### ISSUE TITLE
Helm chart for VPA

### ISSUE THREAD

**Issue by sathieu** [kubernetes/autoscaler#5369]
**Which component are you using?**:

VPA (Vertical Pod Autoscaler)

**Is your feature request designed to solve a problem? If so describe the problem this feature should solve.**:

Easily deploy VPA.

**Describe the solution you'd like.**:

An Helm chart.

**Comment by brsolomon-deloitte** [comment]
Duplicate of https://github.com/kubernetes/autoscaler/issues/3068

**Comment by sathieu** [comment]
Thanks @brsolomon-deloitte

**Comment by gczarnocki** [comment]
There's also: https://artifacthub.io/packages/helm/fairwinds-stable/vpa.

**Comment by Shubham82** [comment]
Hi @jbartosik, your thoughts on this?

**Comment by AhmedGrati** [comment]
Hi @sebastien-prudhomme, wondering what's your stand on moving the chart to this repository and contributing to it here.

**Comment by sebastien-prudhomme** [comment]
Hi @AhmedGrati I try to maintain all the charts I wrote the same way. It will be hard for me to continue the maintenance here while keeping the same consistency. But I can discontinue my chart if someone want to take inspiration from it to build a new one.

The main difference I see with the other Helm chart available in this repo is the fact that I'm using Bitnami common subchart and Bitnami tool to generate the documentation.

**Comment by AhmedGrati** [comment]
Yeah, I asked you because I want to be inspired by your chart to build a new one here. Therefore, I wanted to check whether you have plans to maintain a new chart here or not before I start working on it.

**Comment by sudermanjr** [comment]
I would be willing to consider donating / moving our chart to this repository if it would be a positive impact for the community. https://artifacthub.io/packages/helm/fairwinds-stable/vpa

**Comment by calebAtIspot** [comment]
Can we remove the stale bot from this issue please? It's not a bug that might be randomly fixed by a refactor, and it's already a popular feature request with +18 votes.

**Comment by nicolas-g** [comment]
Let’s close this issue as it duplicates [#3068](https://github.com/kubernetes/autoscaler/issues/3068). It’s better to consolidate the discussion there to keep all updates and traction in one place.

**Comment by adrianmoisey** [comment]
/close

Agreed, please refer to https://github.com/kubernetes/autoscaler/issues/3068
